### PR TITLE
Update/yearly rent debits

### PR DESCRIPTION
**Pull request by dunnudge**
## Link to ticket

Link [here](https://app.clickup.com/t/21phha9)

## Describe this PR
Add endpoint to get yearly rent debits

### CODE REVIEW

**Review by HHjolany — APPROVED** [review]

File: FinanceServicesApi/V1/Controllers/DummyController.cs [hunk 1]
```diff
@@ -44,12 +44,12 @@ public async Task<IActionResult> GetTenures(List<Guid> ids)
             foreach (Guid id in ids)
             {
                 var tenure = await _tenureById.ExecuteAsync(id).ConfigureAwait(false);
-                var housHold = tenure.HouseholdMembers.FirstOrDefault(m =>
+                var houseHold = tenure.HouseholdMembers.FirstOrDefault(m =>
                     m.PersonTenureType == PersonTenureType.Leaseholder
```
Comment (sandipchandra) [inline FinanceServicesApi/V1/Controllers/DummyController.cs:48]: You have to check the Is Responsible flag also here.

File: FinanceServicesApi/V1/UseCase/GetYearlyRentDebitsUseCase.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            if (assetResponse == null)
+            {
+                throw new AssetNotFoundException(assetId, "asset information");
+            }
+
+            var chargesResponse = await _chargesGateway.GetAllByAssetId(assetId).ConfigureAwait(false);
+            var currentTenure = assetResponse.Tenure;
+
+            var tenureInformationResponse = await _tenureInformationGateway.GetById(new Guid(currentTenure.Id)).ConfigureAwait(false);
+            var leaseHolder = tenureInformationResponse?.HouseholdMembers.FirstOrDefault(p => p.IsResponsible);
```
Comment (sandipchandra) [inline FinanceServicesApi/V1/UseCase/GetYearlyRentDebitsUseCase.cs:43]: Yearly Rent Debit is applicable only for Tenants

File: FinanceServicesApi/V1/UseCase/GetYearlyRentDebitsUseCase.cs [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                // Get period. If end date not available default to 52 weeks
+                var tenurePeriodInWeeks = 52m;
+                if (charge == null) continue;
+                if (charge.EndDate >= charge.StartDate)
+                {
+                    tenurePeriodInWeeks = ((charge.EndDate - charge.StartDate).Days) / 7m;
+                }
+                totalAmount += charge.Frequency.ToLower() switch
+                {
+                    "weekly" => charge.Amount * tenurePeriodInWeeks,
```
Comment (sandipchandra) [inline FinanceServicesApi/V1/UseCase/GetYearlyRentDebitsUseCase.cs:83]: Instead of checking the total Weeks count , you can just multiply the value by 52

**Review by sandipchandra — COMMENTED** [review]
Please review the comments

**Pull request merged**